src/Monolog/Handler/Slack/SlackRecord.php

```diff
@@ -115,59 +115,36 @@ public function getSlackData(array $record)
         if ($this->useAttachment) {
             $attachment = array(
                 'fallback' => $message,
+                'text'     => $message,
                 'color'    => $this->getAttachmentColor($record['level']),
                 'fields'   => array(),
             );
 
             if ($this->useShortAttachment) {
                 $attachment['title'] = $record['level_name'];
-                $attachment['text'] = $message;
             } else {
                 $attachment['title'] = 'Message';
-                $attachment['text'] = $message;
-                $attachment['fields'][] = array(
-                    'title' => 'Level',
-                    'value' => $record['level_name'],
-                    'short' => true,
-                );
+                $attachment['fields'][] = $this->generateAttachmentField('Level', $record['level_name'], true);
```

Comment:
I'm sorry for commenting on an outdated diff, will copy my comment on the updated line as it is still valid and so the comment could be visible.